cmd/snap/cmd_info.go

```diff
@@ -163,27 +163,68 @@ func coalesce(snaps ...*client.Snap) *client.Snap {
 	return nil
 }
 
-// formatDescr formats a given string (typically a snap description)
+func rTrimRightSpace(text []rune) []rune {
+	j := len(text)
+	for j > 0 && unicode.IsSpace(text[j-1]) {
+		j--
+	}
+	return text[:j]
+}
+
+func rLastIndexSpace(a []rune) int {
+	for i := len(a) - 1; i >= 0; i-- {
+		if unicode.IsSpace(a[i]) {
+			return i
+		}
+	}
+	return -1
+}
+
+// this is _wrong_ for much of unicode (because the width of a rune on the
+// terminal is anything between 0 and 2, not always 1 as this code assumes)
+// but fixing that is Hard. Long story short, you can get close using a couple
+// of big unicode table. Getting it 100% requires a terminfo-alike of unicode
+// behaviour.
+// This (and probably formatDescr below) should move to strutil once we're
+// happy with it getting wider use.
+func wrap1(out io.Writer, text []rune, width int) {
+	text = rTrimRightSpace(text)
```

Comment:
There's `strings.TrimRightFunc(..)` however I have no clue which version of Go it first appeared in. This would become:
```
strings.TrimRightFunc(string(text), unicode.IsSpace)
```

Reply:
it would become `[]rune(strings.TrimRightFunc(strings(text), unicode.IsSpace)`, which does an encode of text to string, a decode of string to rune, and then another decode of string to rune. Seems wasteful to me.